Pod/Cache/GenericCoreDataStack.swift

```diff
@@ -193,20 +191,6 @@ public class GenericCoreDataStack<CoordinatorType: NSPersistentStoreCoordinator,
                 let storeShmPath = "\(storePath)-shm"
                 let storeWalPath = "\(storePath)-wal"
                 
-                if let requiredProtection = options?[NSPersistentStoreFileProtectionKey] as? String {
-                    //
-                    // If the file protection bit is different than the require protection
-                    // type, we reset the protection.
-                    //
-                    // Note: this must be done before opening the data store either with 
-                    //       metadataForPersistentStoreOfType or normally with
-                    //       addPersistentStoreWithType.
-                    //
-                    try updateFileProtectionIfNeeded(storePath,    requiredProtection: requiredProtection)
```

Comment:
in the future it may be nice to have a method which accepts an array of paths to update